File: Data Structures & Algorithms/sort-colors/submission-0.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int z = 0, o = 0, t = 0, n = nums.size();
        for(int i : nums){
            if(i == 0) z++;
            else if(i == 1) o++;
            else t++;
        }
        int i = 0;
        while(z--){
            nums[i] = 0;
            i++;
        }
        while(o--){
            nums[i] = 1;
            i++;
        }
        while(t--){
            nums[i] = 2;
            i++;
        }
    }
};

File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        s = set()
        lst = []
        for i in nums:
            if i not in s:
                lst.append(i)
                s.add(i)
        for i in range(len(lst)):
            nums[i] = lst[i]
        return len(lst)
        